src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2841,6 +2841,209 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-jparse"
+    title={<InlineCode>jparse(attribute [, path])</InlineCode>}
+  >
+    Use the JSON parse function, `jparse()`, to parse a string value and produce a Map / List of values (or nested structures) which can be handled like any other first-class value type in NRQL.
```

Comment:
We don't generally capitalize data types.

Reply:
Fixed.